### PR TITLE
EmberJS Inspector

### PR DESCRIPTION
**Pull request by joseluisq**
Adds an Ember tab to Chrome DevTools that allows you to inspect Ember objects in your application.
https://chrome.google.com/webstore/detail/ember-inspector/bmdblncegkenkacieihfhpjfppoconhi

### CODE REVIEW

**Comment by paulirish** [comment]
it's a bit wordy. can you shorten the sentence? basically cut it in half or so. (also you'll have to rebase.. sorry)

**Comment by Garbee** [comment]
> Allows you to inspect EmberJS objects in your application.

A much better description here for example. Restating "Chrome DevTools" isn't necessary, since that is all this repository is about. So trim things down to explain just what is added.

**Comment by joseluisq** [comment]
@paulirish sure. @Garbee great! it's more concise !

**Comment by joseluisq** [comment]
PR fixed https://github.com/ChromeDevTools/awesome-chrome-devtools/pull/15
Thanks.
